amethyst_utils/src/removal.rs

```diff
@@ -60,3 +60,16 @@ pub fn exec_removal<I, D>(
         }
     }
 }
+
+/// Adds a `Removal` component with the specified id to the specified entity.
+/// Usually used with prefabs, when you want to add a `Removal` component at the root of the loaded prefab.
+pub fn add_removal_to_entity<T: PartialEq + Clone + Debug + Send + Sync + 'static>(
+    entity: Entity,
+    id: T,
+    storage: &mut WriteStorage<'_, Removal<T>>,
+) {
+    storage.insert(entity, Removal::new(id)).expect(&format!(
```

Comment:
can swap `expect(&format(` with `.unwrap_or_else(|| panic!(`, clippy gives that as a hint i think.

also, since `T: Debug`

```rust
    "Failed to insert `Removal` component ID `{:?}` to entity `{:?}`.",
    id,
    entity
```

Reply:
Fixed.

You can't have id in there without cloning however...